FirebaseAppDistribution/Sources/FIRAppDistributionAuthPersistence.m

```diff
@@ -111,14 +125,18 @@ + (BOOL)persistAuthState:(OIDAuthState *)authState error:(NSError **_Nullable)er
 
 + (NSMutableDictionary *)getKeyChainQuery {
   NSMutableDictionary *keychainQuery = [NSMutableDictionary
-      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass, @"OAuth",
-                                   (id)kSecAttrGeneric, @"OAuth", (id)kSecAttrAccount,
-                                   @"fire-fad-auth", (id)kSecAttrService,
-                                   (id)kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly,
-                                   (id)kSecAttrAccessible, nil];
+      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass,
+                                   kFIRAppDistributionAuthPersistenceErrorKeychainId,
+                                   (id)kSecAttrGeneric, [self BundleIdentifier],
+                                   (id)kSecAttrAccount, kFIRAppDistributionAuthPersistenceServiceId,
+                                   (id)kSecAttrService, nil];
   return keychainQuery;
 }
 
++ (NSString *)BundleIdentifier {
+  return [[NSBundle mainBundle] bundleIdentifier] ?: @"Oauth";
```

Comment:
under what circumstances can this be nil? I ask because we use NSBundle mainBundle bundleIdentifier in FIRAppDistribution.m without verifying it's not nil.

Reply:
good point. fixed.